src/games/strategy/engine/framework/headlessGameServer/HeadlessGameServer.java

```diff
@@ -179,7 +179,6 @@ public String remoteShutdown(final String hashedPassword, final String salt) {
     if (encryptedPassword.equals(hashedPassword)) {
       (new Thread(() -> {
         System.out.println("Remote Shutdown Initiated.");
-        ThreadUtil.sleep(1000);
```

Comment:
And if this is being removed on purpose, the thread running this code is senseless